src/field_theory/separable.lean

```diff
@@ -22,7 +22,7 @@ namespace polynomial
 
 section comm_semiring
 
-variables {R : Type u} [comm_semiring R]
+variables {R : Type u} [comm_semiring R] {S : Type v} [comm_semiring S]
```

Comment:
Could you please mention `expand` and `contract` in the module docstring at the top of this file, under `## Main definitions`.
(I guess you should then also list `separable`, otherwise it looks funny if it is missing.)